source/content/guides/object-cache-pro/01-introduction.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Object Cache Pro
+subtitle: Introduction
+description: 
+tags: []
+contributors: [jazzsequence]
+type: guide
+showtoc: true
+anchorid: object-cache-pro
+permalink: docs/guides/object-cache-pro/
+editpath: object-cache-pro/01-introduction.md
+contenttype: [guide]
+innav: [true]
+categories: []
+cms: [wordpress]
+audience: [development]
+product: [--]
+integration: [--]
+reviewed: ""
```

Comment:
Leaving this empty displays an `Last Reviewed: Invalid date` message. We should add the go-live date or remove entirely. 🔢

Reply:
I noted in the description that there's some stuff, particularly the file header, that should be filled out by the docs team.

Reply:
Perfect, thanks!